pkg/kubelet/kubelet.go

```diff
@@ -705,31 +704,6 @@ func containerRef(pod *api.Pod, container *api.Container) (*api.ObjectReference,
 	return ref, nil
 }
 
-// setRef stores a reference to a pod's container, associating it with the given docker id.
```

Comment:
@vmarmol Does that need to be exported? Are we going to use GenerateContainerRef outside of kubelet package?